from ..exceptions import *
messages = {
    AlreadyJoined: "%(user)s already joined",
    PlayerNotFound: "%(user)s didn't join",
    NotEnoughRights: "you don't have enough rights",
    GameAlreadyStarted: "Game already started",
    UnknownLang: "Unknown Language",
    "lang": "language set to English",
    "lang_help": "set language to English",
    "error": "Internal Error",
    "waiting": "Players:\n%(players)s",
    "new": "New game created",
    "running": "Game is running\n%(state)s\nPlayers:\n%(players)s",
    "join": "%(user)s joined",
    "leave": "%(user)s left",
    "start": "start Game",
    "end": "end Game",
    "open": "Game is now open",
    "close": "Game is now closed",
}
